src/pages/appflow/build-stacks.md

```diff
@@ -7,8 +7,15 @@ nextUrl: "/docs/appflow/web-previews"
 ---
 
 # Build Stacks
+In Appflow build stacks are how we represent the set of software used to build an app on a given platform. This includes the version of npm, Cordova, Ionic, Xcode, and what OS it is all installed on.
```

Comment:
```suggestion
In Appflow, build stacks are how we represent the set of software used to build an app on a given platform. This includes the version of npm, Cordova, Ionic, Xcode, and what OS it is all installed on.
```

If we want to be truly grammatically correct then a comma should be added here.

Reply:
yeah, this is why I like copy editors. commas and I don't get along.

Reply:
Grammarly is my copy editor 😅